utilities/sync_core_libs/ignore_file.py

```diff
@@ -0,0 +1,27 @@
+import os
+from os.path import normpath
+
+from utilities.settings import Settings
+
+
+class IgnoreFile:
+
+    def __init__(self, src_dir):
+        self.ignore_file_name = Settings().get("Ignore_file_name")
```

Comment:
pleace don't use capital letters in settings name.
`"Ignore_file_name"` -> `"ignore_file_name"`